pages/agent/v3/elastic_ci_aws.md.erb

```diff
@@ -43,7 +46,8 @@ Also keep in mind the EC2 HTTP metadata server is available from within builds,
 
 The Buildkite Elastic CI Stack supports:
 
-- All major AWS regions
```

Comment:
I guess major was capturing this but maybe better to be more explicit?

Reply:
Yeah, I don't know what the list is, or what the implications are?